prompter/prompter.go

```diff
@@ -7,9 +9,18 @@ import (
 
 // Prompter prompts users to let them input data, and parses it.
 type Prompter struct {
+	// Writer on which the prompt's label and choices are
+	// written when prompting them.
 	writer io.Writer
+
+	// Reader from which the user's response to the prompt is
+	// read.
 	reader *bufio.Reader
 
+	// Whether or not this prompter is running in a TTY. If this is
```

Comment:
Not exactly, you might want to answer the default value anytime even if you are  in a TTY.

Reply:
Mh 🤔What comment would you suggest then?